docs/style_guide/README.md

```diff
@@ -1,49 +1,48 @@
-# Ballerina style guide
+# Ballerina Style Guide
 
-Ballerina Style Guide aims at maintaining a standard coding style among Ballerina community. Therefore, Ballerina IDE plugins and tools follow this guide when providing code formatting and other features. 
+The Ballerina Style Guide aims at maintaining a standard coding style among the Ballerina community. Therefore, code formatting and the other features provided by the Ballerina IDE plugins and tools are based on this guide. 
```

Comment:
can we change the above sentence to say something like bellow?
```suggestion
The Ballerina Style Guide aims at maintaining a standard coding style among the Ballerina community. Therefore, code formatting tools are based on this guide.
```

Reply:
Fixed.